Dockerfile

```diff
@@ -0,0 +1,21 @@
+# syntax=docker/dockerfile:1
+FROM python:3.7-slim-bullseye
+WORKDIR /
+RUN mkdir -p /opt/virtualenv
+ENV VIRTUAL_ENV="/opt/virtualenv/_installTop_"
```

Comment:
_installTop_ is supposed to be replaced by an actual path. So this line might just be
`ENV VIRTUAL_ENV="/opt/virtualenv"`

Reply:
OK, I'll change this to `/opt/deployutils` and remove the `mkdir` command.

Reply:
We could also standardize around a specific path, like `/app`. That would keep things simple. My only concern with this would be if there is some use-case that makes this confusing or unworkable. Let me know your thoughts on this.

Reply:
So far we haven't had any trouble using `/app`. Note that `/app` is the _installTop_ (for the venv). The source code is copied into `/app/reps/_projectName_`  (i.e.`/app/reps/djaodjin-deployutils`). This way we have a clean directory structure:

- /app/bin
- /app/lib
- /app/reps

I think the `rpmbuild` scripts use `/src` (instead of `/reps`) for the same purpose.

Reply:
Changed!